From 9219e4794cd7963628947a0266a9e96f032fb5e8 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Mon, 26 Oct 2026 21:26:28 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh nettle-release-keyring.gpg from keyservers/WKD: same
certificate(s), with self-signatures our copy was missing.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 mingw-nettle.spec | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/mingw-nettle.spec b/mingw-nettle.spec
index 98a9de8..9557cfb 100644
--- a/mingw-nettle.spec
+++ b/mingw-nettle.spec
@@ -2,7 +2,7 @@
 
 Name:           mingw-nettle
 Version:        3.10.2
-Release:        3%{?dist}
+Release:        4%{?dist}
 
 Summary: MinGW package for nettle cryptographic library
 # Automatically converted from old format: LGPLv3+ or GPLv2+ - review is highly recommended.
@@ -33,6 +33,7 @@ BuildRequires:  mingw64-openssl
 BuildRequires:  autoconf, automake
 BuildRequires:  gcc
 BuildRequires:  m4
+BuildRequires: openpgpverify
 
 
 %description
@@ -71,7 +72,7 @@ or even in kernel space.
 
 %prep
 %autosetup -Tb 0 -p1 -n nettle-%{version}
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 
 
 %build
@@ -132,6 +133,9 @@ rm -rf $RPM_BUILD_ROOT%{mingw64_infodir}/
 
 
 %changelog
+* Fri Oct 23 2026 Jakub Jelen <jjelen@redhat.com> - 3.10.2-4
+- Verify upstream source signatures with openpgpverify
+
 * Thu Jul 16 2026 Fedora Release Engineering <releng@fedoraproject.org> - 3.10.2-3
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_45_Mass_Rebuild
 
-- 
2.56.0

